docs/web-apps/automated-testing/selenium/quickstart.md

```diff
@@ -0,0 +1,217 @@
+
+### Step 1: Clone the Sample Project
+This will clone the project to your machine, and set up your test environment and dependencies.
+
+From your terminal, clone the [Selenium Quickstart Demo Project](https://github.com/saucelabs-training/quickstart-selenium-java) repository to your machine's [home directory](https://en.wikipedia.org/wiki/Home_directory).
+   ```bash
+   git clone https://github.com/saucelabs-training/quickstart-selenium-java.git
+   ```
+
+Or, if you're using an IDE, such as Visual Studio Code or IntelliJ, open the **quickstart-selenium-java** project. The test file is called **SauceTest.java**.
+
+
+### Step 2: Run Test
+From your terminal, execute the test by running:
+```bash
+mvn test
+```
+
+Or, if you're using an IDE, open the `quickstart-appium-java` project you cloned in step 1. Your IDE may ask if you want to enable Java annotation; you can accept this. Next, right-click on the test file (**SauceTest.java**) and choose **Run**.
+
+
+### Step 3: View Test Results on Sauce Labs
+The expected response is a session confirmation message and hyperlink that will take you to your test in progress and [results](/test-results/viewing-test-results/) on Sauce Labs.<br/><img src={useBaseUrl('img/selenium/basic-testresults.png')} alt="basic-testresults" />
+
+
+## Selenium WebDriver Test
+
+In this Selenium test, you'll automate WebDriver to open and close a Chrome browser window.
+
+### Step 1: Clone Project
+From your IDE's terminal, clone the [Sauce Labs Selenium Training](https://github.com/saucelabs-training/advanced-selenium) repository to your home directory.
+  ```bash
+  git clone https://github.com/saucelabs-training/advanced-selenium.git
+  ```
+
+### Step 2: Download Dependencies
+This quickstart test utilizes the Java Development Kit (JDK) ([download here](https://www.oracle.com/java/technologies/downloads/#java8)), which is required to compile and execute Java code, and Selenium Java bindings ([download here](https://www.selenium.dev/downloads/)), which are required for Selenium-Java compatibility.
+
+
+### Step 3: Import Dependencies
+
+1. From your IDE, open the **advanced-selenium** project. The test file is located under **src/test/java/com/saucelabs/advancedselenium/SeleniumTest.java**. <br/><img src={useBaseUrl('img/selenium/selenium-test.png')} alt="selenium-test" />
+2. If it's your first time ever opening a Java project, your IDE will prompt you to set up your JDK. You'll need to:
+   * Click on the prompt.<br/><img src={useBaseUrl('img/selenium/jdk-setup.png')} alt="jdk-setup" width="400"/><br/>
+   * Provide the location where you downloaded JDK. Some IDEs will detect and find the path for you, so that all you have to do is select it.<br/><img src={useBaseUrl('img/selenium/jdk-config.png')} alt="jdk-config" width="500"/>
```

Comment:
```suggestion
   * Provide the location where you downloaded the JDK. Some IDEs will detect and find the path for you, so that all you have to do is select it.<br/><img src={useBaseUrl('img/selenium/jdk-config.png')} alt="jdk-config" width="500"/>
```